src/features/home/HomeProjectInProgress.tsx

```
@@ -0,0 +1,25 @@
+import {userQueries} from '@/entities/user/api';
+
+const HomeProjectInProgress = () => {
+  const {data: projectList} = useQuery(userQueries.groupsAccepted());
+
+  return (
+    <ContentsCard title="진행중인 프로젝트" arrowLink="/projects">
+      <div className="flex flex-col gap-2 mt-[14px] text-left">
+        {projectList && projectList.length > 0 ? (
+          projectList?.map(project => (
+            <div className="p-3.5 rounded-md bg-gray-1 text-gray-dark font-medium text-sm transition hover:bg-primary-light hover:text-primary cursor-pointer">
+              <span>{project.groupName}</span>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)